### DISCUSSION TITLE
Accesing a deployed page on mobile app

### DISCUSSION THREAD

**Discussion by skyisthelimpet** [community/community#50486] [Other Feature Feedback, Questions, & Ideas]
Why can't we go to a deployed website from mobile app?

**Comment by Anmol-Baranwal** [comment]
The GitHub mobile app does not currently support opening hosted websites directly within the app.

The GitHub mobile app is designed to allow users to browse and interact with GitHub repositories and issues, but it does not include a web browser that would allow users to navigate to external websites.

If you want to access a website hosted on GitHub Pages from your mobile device, you can do so by opening a web browser (such as Safari or Chrome) on your device and navigating to the website's URL. Alternatively, you can use a third-party app that supports in-app web browsing to access the website.
